src/codeu/chat/client/commandline/Chat.java

```diff
@@ -27,447 +27,589 @@
+					if (name.equals(userCon.user.name)) {
+						return userCon.user;
+					}
+				}
+				return null;
+			}
+		});
+
+		// U-INT-DEL (remove user interest)
+		//
+		// Add a command that will remove a user from the user's 
+		// interests in the user panel.
+		//   
+		panel.register("u-int-del", new Panel.Command() {
+			@Override
+			public void invoke(List<String> args) {
+				final String name = (args.iterator()).hasNext() ? (args.iterator()).next().trim() : "";
+				if (name.length() > 0) {
+					final User userInterest = findUser(name);
+					if (userInterest == null) {
+						System.out.format("ERROR: No user with name '%s'\n", name);
+					} else {
+						userInterest.interestedUsers.remove(user.user.id);
+					}
+				} else {
+					System.out.println("ERROR: Missing <name>");
+				}
+			}
+
+			//Find user object by name
+			private User findUser(String name) {
+				for (final UserContext userCon : user.users()) {
+					if (name.equals(userCon.user.name)) {
+						return userCon.user;
+					}
+				}
+				return null;
+			}
+		});  
+
+		// C-INT-ADD (add conversation interest)
+		//
+		// Add a command that will add a conversation as an interest
+		// for the user in the user panel.
+		//
+		panel.register("c-int-add", new Panel.Command() {
+			@Override
+			public void invoke(List<String> args) {
+				final String name = (args.iterator()).hasNext() ? (args.iterator()).next().trim() : "";
+				if (name.length() > 0) {
+					final ConversationHeader convoInterest = findConversation(name);
+					if (convoInterest == null) {
+						System.out.format("ERROR: No user with name '%s'\n", name);
+					} else {
+						convoInterest.interestedUsers.add(user.user.id);
+					}
+				} else {
+					System.out.println("ERROR: Missing <title>");
+				}
+			}
+
+			//Find conversation object by name
+			private ConversationHeader findConversation(String title) {
+				for (final ConversationContext conversation : user.conversations()) {
+					if (title.equals(conversation.conversation.title)) {
+						return conversation.conversation;
+					}
+				}
+				return null;
+			}
+		}); 
+
+		// C-INT-DEL (remove conversation interest)
+		//
+		// Add a command that will remove a conversation from the user's 
+		// interests in the user panel.
+		//
+		panel.register("c-int-del", new Panel.Command() {
+			@Override
+			public void invoke(List<String> args) {
+				final String name = (args.iterator()).hasNext() ? (args.iterator()).next().trim() : "";
+				if (name.length() > 0) {
+					final ConversationHeader convoInterest = findConversation(name);
+					if (convoInterest == null) {
+						System.out.format("ERROR: No user with name '%s'\n", name);
+					} else {
+						convoInterest.interestedUsers.remove(user.user.id);
+					}
+				} else {
+					System.out.println("ERROR: Missing <title>");
+				}
+			}
+
+			//Find conversation object by name
+			private ConversationHeader findConversation(String title) {
```

Comment:
maybe we could move this method to the class if it is the same as the findConversation method of the previous c-int-add? (save yourself the handcramp of writing it twice :D )